config/config_test.go

```diff
@@ -378,9 +378,6 @@ func TestValidateForkHeights(t *testing.T) {
 		{
 			"Upernavik", ErrInvalidCfg, "Upernavik is heigher than Vanuatu",
 		},
-		{
-			"Vanuatu", ErrInvalidCfg, "Vanuatu is heigher than Wake",
-		},
```

Comment:
why remove? this test should be valid?

Reply:
reverted, It may be caused by the rebase conflicts resolve